core/src/test/java/io/aiven/klaw/UtilMethods.java

```diff
@@ -180,6 +171,33 @@ public List<UserInfoModelResponse> getUserInfoListModel(String username, String
     return userInfoList;
   }
 
+  public Team getTeamDaoMock() {
+    ServiceAccounts serviceAccounts = new ServiceAccounts();
+    serviceAccounts.setNumberOfAllowedAccounts(-1);
+
+    Team team = new Team();
+    team.setTeamname("Seahorses");
+    team.setTeamId(101);
```

Comment:
For TeamId can we use a different Id, such as 1001 or even 1/2/3 so that we can be sure it is separate from what is being used for the tenantId.